app/src/test/java/org/fossasia/openevent/app/unit/presenter/EventDashboardPresenterTest.java

```diff
@@ -147,7 +147,7 @@ public void shouldShowAttendeeError() {
 
         eventDashboardPresenter.loadDetails(false);
 
-        verify(eventDetailView).showError(Logger.TEST_MESSAGE);
+        verify(eventDetailView).showError(anyString());
```

Comment:
This should not even change. If the throwable is not actually a network error, it should be showing the throwable message